Batch8CucumberSeleniumAPI/src/test/java/pageobjects/SearchPageObjects.java

```diff
@@ -70,6 +67,11 @@ public void ValidateProductList(String productName) {
 				Assert.fail("Product not correctly displayed in the search result. Product at index: " + (i+1));
 			}
 		}
-		
+	}
+	public void ValidateInvalidateProductMessage(){
+		String expected="Try Checking Your Spelling Or Use More General Term.";
```

Comment:
Accept the argument, this method should look like this.
`public void ValidateInvalidateProductMessage(String errMessage){`
`String actual=getText(result_invalid_product);
		Assert.assertEquals(errMessage, actual);
		scn.write("Error message is displayed");`